examples/with-vercel-kv/app/page.tsx

```diff
@@ -1,4 +1,4 @@
-import { Ratelimit } from "@upstash/ratelimit";
+import { Ratelimit } from "../../../src";
```

Comment:
I still believe we need a more elaborate example that incorporates OpenAI or any other AI provider.

Reply:
Yes I have to create a separate example for it

Reply:
You rock 🤘